lib/codegen/generator.dart

```diff
@@ -292,7 +292,7 @@ class _DeclarationCode extends ConstExpression {
 
   List<String> get librariesUsed => []
     ..addAll(type.librariesUsed)
-    ..addAll(annotations.expand((a) => a.librariesUsed));
+    ..addAll(annotations.expand((a) => a.librariesUsed) as Iterable<String>);
```

Comment:
It would be nice if `expand` could take a generic type, but since that isn't the case today this is fine.